composer.json

```diff
@@ -185,6 +185,7 @@
         "psr-4": {
             "Symfony\\Bridge\\Doctrine\\": "src/Symfony/Bridge/Doctrine/",
             "Symfony\\Bridge\\Monolog\\": "src/Symfony/Bridge/Monolog/",
+            "Symfony\\Bridge\\PhpUnit\\": "src/Symfony/Bridge/PhpUnit/",
```

Comment:
this should be reverted

Reply:
Unit tests fail without it.

Reply:
Locally or with GitHub Actions?

Reply:
Both, eg: https://github.com/HypeMC/symfony/actions/runs/12646955119

Reply:
this will be fixed by merging IIRC: classes will be loaded from the installed phpunit-bridge

Reply:
@nicolas-grekas Oh, I get it now, removed.